content/en/users/cloud-compute/faq/_index.md

```diff
@@ -1,43 +1,65 @@
 ---
 title: "FAQ"
 type: docs
-weight: 120 
+weight: 120
 description: >
-  Frequenlty Asked Questions 
+  Frequenlty Asked Questions
 ---
 
 ## Basics
 
 ### How can I get access to the EGI.eu Federated Cloud?
```

Comment:
`EGI.eu` ?

Reply:
wow, no idea how that one managed to get there